kubetest2/internal/deployers/eksapi/deployer.go

```diff
@@ -189,7 +189,7 @@ func (d *deployer) Up() error {
 	if err != nil {
 		return err
 	}
-	if d.deployerOptions.StaticClusterName != "" || d.deployerOptions.AutoMode {
+	if d.deployerOptions.StaticClusterName != "" {
```

Comment:
This was a bug -- the `staticClusterManager` was likely `nil` if `--auto-mode` was used.